client/my-sites/google-my-business/select-location/location.js

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+const GoogleMyBusinessLocation = ( {
+	address,
+	name,
+	photo,
+	siteSlug,
+	translate,
+	verified,
+} ) => (
+	<CompactCard className="gmb-location">
+		<div className="gmb-location__content">
+			<img
+				src={ photo ? photo : '/calypso/images/google-my-business/spacer.png' }
+				className="gmb-location__image"
+				alt={ translate( 'Profile picture' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **87** times:
`translate( 'Profile Picture' )` *ES Score: 20*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).